src/main/java/com/minecolonies/coremod/items/ModItems.java

```diff
@@ -34,4 +36,39 @@ private ModItems()
          * Intentionally left empty.
          */
     }
-}
+
+    /**
+     * Initates all the blocks. At the correct time.
+     * @param registry
+     */
+    public static void init(final IForgeRegistry<Item> registry)
+    {
+        supplyChest = new ItemSupplyChestDeployer();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Synchronize this lazy initialization of 'supplyChest' [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS2444)